runtime.md

```diff
@@ -33,24 +33,79 @@ This is provided so that consumers can find the container's configuration and ro
 }
 ```
 
-## Lifecycle
+## Operations
+
+OCI compliant runtimes MUST support the following operations, unless support for the operation can not be supported by the base operating system (kernel) itself.
+
+OPEN: we should consider specifying the options/args for each operation.
+For example, checkpoint (for runc) seems to have a lot of params.
 
 ### Create
 
-Creates the container: file system, namespaces, cgroups, capabilities.
+Using the `config.json` and `runtime.json` files, along with the file system path referenced in the `config.json`, this operation MUST create a new container.
+This includes creating the relevant namespaces, cgroups and configuring the appropriate capabilities for the container.
+
+### Start
+
+This operation MUST start a previously created container.
```

Comment:
I'm personally in favor of having distinct “create” and “start” steps, but [we don't have that in runC now][1].  I'm in favor of [just echoing runC's approach now to get some lifecycle docs landed][2], after which we can have more focused discussions about things like this.

[1]: https://groups.google.com/a/opencontainers.org/d/msg/dev/frUXLljXy8Y/8pH-81moGgAJ
[2]: https://github.com/opencontainers/specs/pull/207#discussion_r41798699